test/cypress/integration/login.js

```diff
@@ -71,4 +71,12 @@ describe('Login helpers', () => {
     cy.cookieLogin(adminUsername, adminPassword);
     cy.contains(adminUsername);
   });
+
+  it('can use apiLogin', () => {
+    cy.apiLogin('admin', 'admin');
```

Comment:
```suggestion
    cy.apiLogin(adminUsername, adminPassword);
```

(because these are configurable)

Reply:
Ok.